cmd/src/server.rs

```diff
@@ -322,12 +324,36 @@ impl TiKVServer {
         let region_info_accessor = RegionInfoAccessor::new(&mut coprocessor_host);
         region_info_accessor.start();
 
+        let yatp_read_pool = if self.config.readpool.unify_read_pool {
+            let unified_read_pool_cfg = &self.config.readpool.unified;
+            let mut builder = yatp::Builder::new("yatp-read-pool");
+            builder
+                .min_thread_count(unified_read_pool_cfg.min_thread_count)
+                .max_thread_count(unified_read_pool_cfg.max_thread_count)
+                .max_inplace_spin(unified_read_pool_cfg.max_inplace_spin);
```

Comment:
I think `max_inplace_spin` makes more sense to `runner` instead of thread pool. It may be removed in the future as `SpinWait` is used now.

Reply:
Do you think we can remove this configuration? Once we give it a good default value, I think it needn't be changed by conf.

Reply:
Yes. I will clean up the configuration and only keep those that are actually taking effects.